add pages for building enterprise consent journey (#879)
codatio/codat-docs: docs/enterprise/tech-implementation/consent/considerations.md

---
title: "Considerations"
description: "Things to take into consideration when building and deploying a consent journey"
---

#### **Security & Data Privacy**

> In our agreement with you, Codat acts as the Processor of your customers data, you will remain the controller. This relationship allows you to maintain full control over your customers data, including when deciding when to delete your customers data. 

Reviewer: ```suggestion
In our agreement with you, Codat acts as the Processor of your customers' data while you will remain its controller. This relationship allows you to maintain full control over your customers' data, including deciding when to delete that data. 
```

You will likely want to provide the customer with an overview of your Security & Data Privacy policies within your strategic consent journey.

#### **Manage Connections UI**

The customer should have the ability to add, change or delete existing connections via a UI. Codat’s API can be used to add additional integrations or delete existing integrations.

It is critical that customers are aware of potential impacts to other use-cases or products when deleting connections (I.e. Multi-use cases or products)

#### **“Will you have access to all my data?”**

The client journey will include direct contact with [CB Markets FX telesales team] who will typically be on a call to the client when they share the link.  Once the client receives it, they can log in through the accountancy platform integration provided by Codat.  

> Their data is encrypted in transit using SSL (Secure Socket Layer) and at rest using AES-256 (Advanced Encryption Standard - 256 bit). These robust protocols are a requirement of GDPR and are industry standard for processes such as Open Banking. 

Reviewer: ```suggestion
Their data is encrypted using Secure Socket Layer (SSL) in transit and Advanced Encryption Standard - 256 bit (AES-256) at rest. These robust protocols are a requirement of GDPR and are industry standard for processes such as Open Banking. 
```

This question should be added into a wider set of FAQs that are surfaced within the Strategic Consent Journey.

#### **“Will my data be shared securely?”**

A customer will likely be unsure about  the underlying technology that allows data to be extracted & surfaced into each relevant application via APIs. 

> It is important to provide some comfort to each customer connecting banking &/or accounting packages that their data will be shared very securely via 0Auth2.0.

Reviewer: ```suggestion
It is important to provide some comfort to each customer connecting banking and/or accounting packages that their data will be shared very securely via 0Auth2.0.
```

This question should be added into a wider set of FAQs that are surfaced within the Strategic Consent Journey.

#### **“What if my accounting package isn’t listed?”**

A customer might use an accounting package that isn’t covered by Codat’s set of integrations. If this is the case, it is important to provide guidance for the customer. 

The alternative process will differ for each product, but might involve uploading or sharing data manually.

This question should be added into a wider set of FAQs that are surfaced within the Strategic Consent Journey.

#### **“Will you share my data with any third parties”**

> A customer might be concerned that their data will be used for other reasons other than the specific product or journey they are connecting via.

Reviewer: ```suggestion
A customer might be concerned that their data will be used for reasons other than the specific product or journey they are connecting via.
```

It is important to provide the customer with reassurance that their data is only being used for the specific purpose within each user journey.

This question should be added into a wider set of FAQs that are surfaced within the Strategic Consent Journey.

#### **“How do I revoke access”?**

> A customer will want to know how to revoke access, which will be possible via the Strategic Consent Journey UI - as well as the underlying accounting package.

Reviewer: ```suggestion
A customer will want to know how to revoke access, which is possible via the Strategic Consent Journey UI as well as the underlying accounting package.
```

This question should be added into a wider set of FAQs that are surfaced within the Strategic Consent Journey.

#### **GDPR**

> Codat acts as the Processor of your client data, whilst the you remains the controller. This relationship allows the you to maintain full control over the client’s data, including when deciding when to delete it.  There are links to the your Security & Data Privacy Policy within this consent journey.

Reviewer: ```suggestion
Codat acts as the Processor of your client data, while you remain its controller. This relationship allows you to maintain full control over the clients' data, including deciding when to delete it.  There are links to your Security & Data Privacy Policy within this consent journey.
```
